### PR TITLE
Merge artworks in crawler

### PR DESCRIPTION
**Pull request by JaGennis**
Fixes issue [74](https://github.com/hochschule-darmstadt/openartbrowser/issues/74)

### CODE REVIEW

File: scripts/Wikidata crawler/ArtOntologyCrawler.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import json
 
 DEV = False
-DEV_LIMIT = 5
+DEV_LIMIT = 50
```
Comment (tilomi) [inline scripts/Wikidata crawler/ArtOntologyCrawler.py]: remove please the rest looks right to me

Comment (JaGennis) [reply scripts/Wikidata crawler/ArtOntologyCrawler.py]: done

**Comment by tilomi** [comment]
I'll test it before we merge. We were unsure if some fields could be missing.

**Comment by tilomi** [comment]
There is still a bug with creating the json from the paintings.csv, sculptures.csv, drawings.csv. The n-ary attributes (like creators) are written to a string instead of an array of values

**Comment by tilomi** [comment]
This is working now. It should be checked again and can be merged afterwards

**Comment by jschurse** [comment]
I checked it and can verify that the extraction and merging functionality works as expected.

**Pull request merged**